src/async_fuse/memfs/s3_metadata.rs

```diff
@@ -550,11 +550,11 @@ impl<S: S3BackEnd + Sync + Send + 'static> MetaData for S3MetaData<S> {
             Some(ref path) => Some(path.as_ref()),
             None => None,
         };
-        let uid = param.uid;
-        let gid = param.gid;
+        let user_id = param.uid;
```

Comment:
I think `uid` and `gid` is accurate in this case. The naming is a kind of convention.